drivers/volume/portworx/portworx.go

```diff
@@ -968,7 +968,15 @@ func (d *portworx) WaitDriverUpOnNode(n node.Node, timeout time.Duration) error
 		logrus.Debugf("checking PX status on node: %s", n.Name)
 		pxNode := nodeInspectResponse.Node
 		switch pxNode.Status {
-		case api.Status_STATUS_DECOMMISSION, api.Status_STATUS_OK: // do nothing
+		case api.Status_STATUS_DECOMMISSION: // do nothing
+		case api.Status_STATUS_OK:
+			err := d.testAndSetEndpointUsingNodeIP(pxNode.MgmtIp)
+			if err != nil || strings.Contains(err.Error(), "connect: connection refused") {
```

Comment:
err != nil check should be enough? Is the strings.Contains() check even valid if err == nil?